_posts/2023-05-24-cryostat-release.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Cryostat 2.3.0 is Released!
+date: 2023-05-23
```

Comment:
Date is titled for `2023-05-23`  but file is named `2023-05-24-...`.

Reply:
Oh, I think I just set the date like that so the post would appear on the dev build of the site while I was writing it/it was in review. I will update this and the filename to correspond to whatever the date is when this is going to be merged and go live.

Reply:
I think the `future` flag in the `bundle exec jekyll serve --watch --livereload --open-url --future --config _config.yml,_config-dev.yml` should account for future posts right? :-)